d2bs/kolbot/libs/OOG.js

```diff
@@ -900,18 +901,49 @@ MainLoop:
 			delay(25);
 		}
 
-		if (getLocation() === 12) {
+		// start from beginning of the char list
+		sendKey(0x24);
+
+		while (getLocation() === 12 && count < 24) {
 			control = getControl(4, 37, 178, 200, 92);
 
 			if (control) {
 				do {
 					text = control.getText();
 
-					if (text instanceof Array && typeof text[1] === "string" && text[1] === info.charName) {
-						return true;
+					if (text instanceof Array && typeof text[1] === "string") {
+						count++;
+
+						if (text[1] === info.charName) {
+							return true;
+						}
+
+						if (count >= 24) {
```

Comment:
consider moving to while loop:
`} while (count < 24 && control.getNext());`

Reply:
fixed as requested